src/test/unit/views/live-case-status.test.ts

```diff
@@ -0,0 +1,75 @@
+import { expect } from 'chai';
+import request from 'supertest';
+
+import { app } from '../../../main/app';
+
+const PAGE_URL = '/live-case-status?courtId=1';
+const expectedHeader = 'Live hearing updates - daily court list';
+const expectedCourtName = 'Mutsu Court';
+let htmlRes: Document;
+
+describe('Live Status page', () => {
+  beforeAll(async () => {
+    await request(app).get(PAGE_URL).then(res => {
+      htmlRes = new DOMParser().parseFromString(res.text, 'text/html');
+    });
+  });
+
+  it('should display a back button with the correct value', () => {
+    const backLink = htmlRes.getElementsByClassName('govuk-back-link');
+    expect(backLink[0].innerHTML).contains('Back', 'Back button does not contain correct text');
+    expect(backLink[0].getAttribute('href')).equal('#', 'Back value does not contain correct link');
```

Comment:
this doesnt really apply if youre using history.back, so can we change the test to match?